litmus-portal/frontend/src/utils/validate.ts

```diff
@@ -60,7 +60,7 @@ export const validateLength = (value: string) => {
 };
 
 export const isValidWebUrl = (value: string) => {
-  const regEx = /^https:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$/gm;
+  const regEx = /^(http|https):\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$/gm;
   if (value.match(regEx) || value.length === 0) return true;
```

Comment:
0 is a falsy value you can use just `value.match(regEx) || value.length` in the if condition